youtube_dl/extractor/abc.py

```diff
@@ -103,6 +103,65 @@ def _real_extract(self, url):
         }
 
 
+class ABCIViewShowIE(InfoExtractor):
+    IE_NAME = 'abc.net.au:iview:show'
+    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*show/(?P<id>[^/?#]+)/?'
+    _GEO_COUNTRIES = ['AU']
+
+    # ABC iview programs are normally available for 14 days only.
+    _TESTS = [{
+        'url': 'https://iview.abc.net.au/show/sarah-and-duck/',
+        'info_dict': {
+            'id': 'sarah-and-duck',
+            'title': 'Sarah And Duck',
+            'description': 'Sarah is a 7 year old girl with big eyes and a green hat, who lives with her quacky, flappy, slightly manic, but endearing best friend, Duck.',
+        },
+        'playlist_count': 14,
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+@classmethod
+def suitable(cls, url):
+    return False if ABCIViewIE.suitable(url) else super(ABCIViewShowIE, cls).suitable(url)
```

Comment:
```suggestion
    @classmethod
    def suitable(cls, url):
        return False if ABCIViewIE.suitable(url) else super(ABCIViewShowIE, cls).suitable(url)
```

Apologies, this suggestion caused the tests to fail because it's not indented correctly. I've added 4 spaces to each line to fix it.

I'm unable to add a multi-line suggestion so you might need to fix this up manually.

Reply:
Fixed